src/components/Pill/PillButton/PillButton.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import { IoIosArrowUp, IoIosArrowDown } from 'react-icons/io';
+import { CROSS_CHAR, ENTER_KEY } from '../../../constants';
+import styles from './PillButton.scss';
+
+const { block, elem } = bem({
+    name: 'PillButton',
+    classnames: styles,
+    propsToMods: ['isActive', 'isOpen'],
+});
+
+const PillButton = React.forwardRef((props, ref) => {
+    const { isOpen, togglePopup, onClear, name, label, ...rest } = props;
+    const isActive = !!label;
+    const propsForBem = { ...props, isActive };
+
+    let buttonIcon;
+    let useButtonClick = false;
```

Comment:
maybe we should consider renaming this variable because now it can be mixed up with hook?